### DISCUSSION TITLE
kidzone tv [ee]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#893] [Q&A]
### Channel ID

KidZoneMax.ee

### Notes (optional)

it airs shows in other languages than just estonian

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by DJ1999-max** [comment]
Please add stream. Thank you!

**Comment by dantheman121** [comment]
> Please add stream. Thank you!

I don’t have a stream

**Comment by DJ1999-max** [comment]
Sad, task is done.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19873
